### ISSUE TITLE
long title can unnecessarily push power number off the emblem when the header is narrow

### ISSUE THREAD

**Issue by nev-r** [DestinyItemManager/DIM#10289]
![image](https://github.com/DestinyItemManager/DIM/assets/68782081/ccdf6969-51b7-4ffa-af38-59832fb3fd8f)

**Comment by Darkoberd00** [comment]
Hi,

I have found a way to fix this error. Without neglecting the mobile version, just like the D1 version, but unfortunately with the loss of the complete text.

In the SCSS class "class" you used an `overflow: hidden` for the text as well as `text-overflow: ellipsis`, I have now added this to the class "bottom" as well.

I can also make a pull request for the changed file.

![Screenshot from 2024-03-21 16-03-40](https://github.com/DestinyItemManager/DIM/assets/24703416/bc50790a-8113-4811-86fb-8890fdb337e2)
